sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogOptions.java

```diff
@@ -202,12 +207,60 @@ public boolean isPrettyPrintBody() {
     /**
      * Sets flag to allow pretty printing of message bodies.
      *
-     * @param prettyPrintBody If true, pretty prints message bodies when logging. If the detailLevel does not
-     *                        include body logging, this flag does nothing.
+     * @param prettyPrintBody If true, pretty prints message bodies when logging. If the detailLevel does not include
+     * body logging, this flag does nothing.
      * @return The updated HttpLogOptions object.
      */
     public HttpLogOptions setPrettyPrintBody(boolean prettyPrintBody) {
         this.prettyPrintBody = prettyPrintBody;
         return this;
     }
+
+    /**
+     * Gets the {@link HttpRequestLogger} that will be used to log requests.
+     * <p>
+     * A default logger will be used if one isn't supplied.
+     *
+     * @return The {@link HttpRequestLogger} that will be used to log requests.
+     */
+    public HttpRequestLogger getRequestLogger() {
+        return requestLogger;
+    }
+
+    /**
+     * Sets the {@link HttpRequestLogger} that will be used to log requests.
+     * <p>
+     * A default logger will be used if one isn't supplied.
+     *
+     * @param requestLogger The {@link HttpRequestLogger} that will be used to log requests.
+     * @return The updated HttpLogOptions object.
+     */
+    public HttpLogOptions setRequestLogger(HttpRequestLogger requestLogger) {
+        this.requestLogger = requestLogger;
+        return this;
+    }
+
+    /**
+     * Gets the {@link HttpResponseLogger} that will be used to log responses.
+     * <p>
+     * A default logger will be used if one isn't supplied.
+     *
+     * @return The {@link HttpResponseLogger} that will be used to log responses.
+     */
+    public HttpResponseLogger getResponseLogger() {
+        return responseLogger;
+    }
+
+    /**
+     * Sets the {@link HttpResponseLogger} that will be used to log responses.
+     * <p>
+     * A default logger will be sued if one isn't supplied.
```

Comment:
nit; sued -> used

Reply:
Good catch, I'll take a good look through the new Javadocs.